tests/test-sources/plugins/by-name/timerly/default.nix

```
@@ -0,0 +1,26 @@
+{
+  empty = {
+    plugins = {
+      timerly.enable = true;
+    };
+  };
+
+  defaults = {
+    plugins.timerly.enable = true;
```

```suggestion
    plugins.timerly = {
      enable = true;
      
      settings = {
        minutes = [ 25 5];
        ...
      };
    };
```
fill with https://github.com/nvzone/timerly/blob/17299a4d332c483ce09052fe8478b41b992f2594/lua/timerly/state.lua#L12-L21

---

(this is optional)

The main reason to explicitly define the plugin's defaults in a `defaults` test case is to ensure that any settings-options we declare are valid (or somewhat valid).

This doesn't have much value in a plugin that doesn't declare any settings-options :wink:

---

yeah I thought that the empty test case would be enough at least as far as defaults go

---

I ended up turning example into a mix of the defaults and new settings, I think this would be good enough as far as tests go for this plugin